News_Blockchain/BlockValidator.cs

```diff
@@ -244,36 +245,64 @@ private bool CheckCoinbaseTransaction(Block block, int height)
         /// <param name="pubkey"></param>
         /// <param name="senderPublickKey"></param>
         /// <returns>true or false</returns>
-        private bool CheckTransactionSignature(Transacation_Input transaction, Transacation_Output transaction_Output, string pubkey, string senderPublickKey, string signature)
+        private bool CheckTransactionSignature(Transacation_Input transaction, Transacation_Output transaction_Output, string pubkey, string senderPublickKey)
         {
             string pubkeyCoppy = pubkey;
 
             string pubkeyHash = Helpers.ComputeSHA256Hash(pubkeyCoppy, 2);
             string publickKeyHash = Helpers.ComputeSHA256Hash(senderPublickKey, 2);
-            string signatureHash = Helpers.ComputeSHA256Hash(transaction.stringSignature, 2);
 
-            if (pubkeyHash != publickKeyHash)
-                return false;
+            byte[] data = Encoding.ASCII.GetBytes("message");
+            byte[] hash = System.Security.Cryptography.SHA256.Create().ComputeHash(data);
+
+            RSAParameters sharedParameters;
```

Comment:
Use ECC not RSA, take a look at https://billatnapier.medium.com/ecdsa-and-c-5cce146335c9